Library/Homebrew/cmd/update-report.rb

```diff
@@ -126,131 +127,137 @@ def output_update_report
 
       updated = true
 
-      old_tag = Settings.read "latesttag"
+      old_repository_version = Settings.read "latesttag"
 
-      new_tag = Utils.popen_read(
+      new_repository_version = Utils.popen_read(
         "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
       ).lines.first.chomp
 
-      if old_tag.blank? || (new_tag == old_tag)
+      if old_repository_version.blank? || (new_repository_version == old_repository_version)
         puts "Updated Homebrew from #{shorten_revision(initial_revision)} to #{shorten_revision(current_revision)}."
       else
-        new_repository_version = new_tag
-        puts "Updated Homebrew from #{old_tag} (#{shorten_revision(initial_revision)}) " \
-             "to #{new_tag} (#{shorten_revision(current_revision)})."
+        Settings.write "latesttag", new_repository_version
+        puts "Updated Homebrew from #{old_repository_version} (#{shorten_revision(initial_revision)}) " \
+             "to #{new_repository_version} (#{shorten_revision(current_revision)})."
       end
     end
 
     Homebrew.failed = true if ENV["HOMEBREW_UPDATE_FAILED"]
-    return if Homebrew::EnvConfig.disable_load_formula?
-
-    hub = ReporterHub.new
-
-    updated_taps = []
-    Tap.each do |tap|
-      next unless tap.git?
-      next if (tap.core_tap? || tap == "homebrew/cask") && Homebrew::EnvConfig.install_from_api?
-
-      if ENV["HOMEBREW_MIGRATE_LINUXBREW_FORMULAE"].present? && tap.core_tap? &&
-         Settings.read("linuxbrewmigrated") != "true"
-        ohai "Migrating formulae from linuxbrew-core to homebrew-core"
-
-        LINUXBREW_CORE_MIGRATION_LIST.each do |name|
-          begin
-            formula = Formula[name]
-          rescue FormulaUnavailableError
-            next
+    unless Homebrew::EnvConfig.disable_load_formula?
```

Comment:
I think it's worth just leaving this as-is.

Reply:
I will remove the commit.